src/ILCompiler.Compiler/src/IL/ILImporter.Scanner.cs

```diff
@@ -406,6 +406,12 @@ private void ImportCall(ILOpcode opcode, int token)
 
             TypeDesc exactType = method.OwningType;
 
+            if ((method.Signature.IsStatic && reason == "callvirt")
```

Comment:
Could you use `opcode` instead of `reason` here? The string comparison unnecessarily slows things down here.